.travis.yml

```diff
@@ -12,7 +12,7 @@ before_install:
 
 install:
   - npm install
-  - npm install cucumber
+  - npm install cucumber@1.0.0
```

Comment:
@simondean The code works only with cucumber 1.0.0, because in 1.1.0 they introduced a breaking change (AstTreeWalker was completely rewritten).

I'll update the npm install version to use '~1.0.0'